std_msgs/Header header

#线的两端的点
geometry_msgs/Point32 begin_p
geometry_msgs/Point32 end_p